### ISSUE TITLE
NSInternalInconsistencyException__NSCFSet

### ISSUE THREAD

**Issue by SudoPlz** [Instabug/Instabug-iOS#183]
Hey there.

Here's one from today.

```
NSInternalInconsistencyException -[__NSCFSet removeAllObjects]: mutating method sent to immutable object 
    Frameworks/CoreFoundation.framework/CoreFoundation ___exceptionPreprocess
    /usr/lib/libobjc.A.dylib objc_exception_throw
    Frameworks/CoreFoundation.framework/CoreFoundation +[NSException raise:format:]
    Frameworks/CoreFoundation.framework/CoreFoundation -[__NSCFSet removeAllObjects]
    Frameworks/CoreData.framework/CoreData -[NSManagedObjectContext(_NSInternalChangeProcessing) _propagateDeletesUsingTable:]
    Frameworks/CoreData.framework/CoreData -[NSManagedObjectContext(_NSInternalChangeProcessing) _processDeletedObjects:]
    Frameworks/CoreData.framework/CoreData -[NSManagedObjectContext(_NSInternalChangeProcessing) _propagatePendingDeletesAtEndOfEvent:]
    Frameworks/CoreData.framework/CoreData -[NSManagedObjectContext(_NSInternalChangeProcessing) _processRecentChanges:]
    Frameworks/CoreData.framework/CoreData -[NSManagedObjectContext save:]
    Instabug/Logging/IBGEventLogger.m:50:9 __45-[IBGEventLogger addEventLogWithName:params:]_block_invoke
    Instabug/IBGDALService.m:51:17 __44-[IBGDALService writeASync:completionBlock:]_block_invoke_3
    Frameworks/CoreData.framework/CoreData developerSubmittedBlockToNSManagedObjectContextPerform
    /usr/lib/system/libdispatch.dylib <redacted>
    /usr/lib/system/libdispatch.dylib <redacted>
    Frameworks/CoreData.framework/CoreData -[NSManagedObjectContext performBlockAndWait:]
    Instabug/IBGDALService.m:50:13 __44-[IBGDALService writeASync:completionBlock:]_block_invoke_2
    /usr/lib/system/libdispatch.dylib <redacted>
    /usr/lib/system/libdispatch.dylib <redacted>
    Instabug/IBGDALService.m:68:13 __44-[IBGDALService writeASync:completionBlock:]_block_invoke
    /usr/lib/system/libdispatch.dylib <redacted>
    /usr/lib/system/libdispatch.dylib <redacted>
    /usr/lib/system/libdispatch.dylib <redacted>
    /usr/lib/system/libdispatch.dylib <redacted>
    /usr/lib/system/libdispatch.dylib <redacted>
    /usr/lib/system/libdispatch.dylib <redacted>
    /usr/lib/system/libdispatch.dylib <redacted>
    /usr/lib/system/libsystem_pthread.dylib __pthread_wqthread
    /usr/lib/system/libsystem_pthread.dylib start_wqthread
```

**Comment by Kmohamed** [comment]
@SudoPlz This gonna be fixed in the next release.

**Comment by Kmohamed** [comment]
@SudoPlz This issue is fixed on the latest version 7.3.6.
